plugins/macos.yaml

```diff
@@ -0,0 +1,109 @@
+# Plugin Info
+version: 0.0.1
+title: MacOS
+description: Log parser for MacOS
+parameters:
+  - name: enable_system_log
+    label: System Logs
+    description: Enable to collect MacOS system logs
+    type: bool
+    default: true
+  - name: system_log_path
+    label: System Log Path
+    description: The absolute path to the System log
+    type: string
+    default: "/var/log/install.log"
+  - name: enable_install_log
+    label: Install Logs
+    description: Enable to collect MacOS install logs
+    type: bool
+    default: true
+  - name: install_log_path
+    label: Install Log Path
+    description: The absolute path to the Install log
+    type: string
+    default: "/var/log/system.log"
+  - name: start_at
+    label: Start At
+    description: Start reading file from 'beginning' or 'end'
+    type: enum
+    valid_values:
+      - beginning
+      - end
+    default: end
+
+# Set Defaults
+# {{$enable_system_log := default true .enable_system_log}}
+# {{$system_log_path := default "/var/log/system.log" .system_log_path}}
+# {{$enable_install_log := default true .enable_install_log}}
+# {{$install_log_path := default "/var/log/system.log" .install_log_path}}
+# {{$start_at := default "end" .start_at}}
+
+# Pipeline Template
+pipeline:
+  # {{ if $enable_system_log }}
+  - id: system_input
+    type: file_input
+    include:
+      - {{ $system_log_path }}
+    start_at: {{ $start_at }}
+    multiline:
+      line_start_pattern: '\w{3} \d{1,2} \d{2}:\d{2}:\d{2}'
+    labels:
+      log_type: macos.system
+      plugin_id: {{ .id }}
+    output: system_parser
+
+  - id: system_parser
+    type: regex_parser
+    regex: '^(?P<timestamp>\w{3} \d{1,2} \d{2}:\d{2}:\d{2}) (---|(?P<host>[^ ]*))? ((?P<process>[^\[]*)\[(?P<pid>[^\]]*)\])?( \((?P<subprocess>[^\[]*)(\[(?P<spid>[^\]]*)\])?\))?(: )?(?P<message>[\w\W]*)'
+    timestamp:
+      parse_from: timestamp
+      layout_type: gotime
+      layout: 'Jan _2 15:04:05'
```

Comment:
Does this format always exist?

Reply:
It existed in my log for a brief period of time in the middle of my log. It then switch back to timestamp format that seems to be the norm.  I am unsure it if this format shows up everywhere, but it caused errors with parsing my log file. It shouldn't hurt to handle it even it is just unique to my log.